### ISSUE TITLE
[FEATURE REQUEST] Automated graphical detail level through benchmarking

### ISSUE THREAD

**Issue by jebeld17** [supertuxkart/stk-code#3743]
## Description:
In order to ensure quality of gameplay for the vast majority of users, it is arguable that FPS should be the primary thing saught after. Super TuxKart does not currently have any in-place benchmark to determine the correct graphics setting to run as smoothly as possible on the device, though.
My request in this post is to add such a feature to the program. 

## Configuration:
STK release version: 0.9.3
STK Source (PPA, distribution package, official binary, etc.): https://play.google.com/store/apps/details?id=org.supertuxkart.stk
System: HTC U11 128GB(USA Unlocked)
Graphics card: Adreno 540
CPU: Qualcomm Snapdragon 835
Gamepads/keyboards models if related to the issue: On-screen

**Comment by Alayan-stk-2** [comment]
I agree this would be a nice feature to have to help those users who don't think of adapting the graphics settings themselves to reach their preferred quality/resolution/FPS balance; or who may test them on a less demanding track before seeing worse performance in a more demanding one.

But this will not be high priority to implement.

**Comment by jebeld17** [comment]
How about [also] suggesting settings based off benchmarks performed on a computer with like-specs for those who have chosen not to run the benchmark?

**Comment by auriamg** [comment]
Doing benchmarks or comparing computers is extremely tricky and unpredictable. If we attempted this, we'd most likely end up with an approximation that's too coarse to give truly useful results, unless a very significiant amount of work was invested
